molgenis-data-annotators/pom.xml

```diff
@@ -43,11 +43,6 @@
 			<artifactId>spring-webmvc</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.httpcomponents</groupId>
-			<artifactId>httpclient</artifactId>
-			<version>4.2.1</version>
-		</dependency>
-		<dependency>
```

Comment:
verified that the annotators still work, probably an artifact from the late chembl annotator